.github/workflows/build-cli.yml

```diff
@@ -0,0 +1,75 @@
+name: Build CLI
+
+on:
+  push:
+    branches: [main]
```

Comment:
Should we do every push to main, or just nightlies?

Reply:
it's not _that_ expensive, and is nice at least while we're rapidly changing things to be able to ship updated wheels?